<launch>
  <!-- vim: set ft=xml noet : -->
  <!-- base node launch file-->

  <arg name="address" default="192.168.50.110" /> 
   
  <node pkg="web_video_server" type="web_video_server" name="web_video_server" output="screen">
    <param name="port" type="int" value="8020" />
    <param name="address" type="string" value="$(arg address)" />

    <param name="server_threads" type="int" value="1" />
    <param name="ros_threads" type="string" value="2" />

    <!--param name="width" type="int" value="1280" /-->
    <!--param name="height" type="int" value="1080" /-->
    <!--param name="quality" type="int" value="90" /-->
    <!--param name="height" type="int" value="1080" /-->
  </node>

</launch>
